apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/post/PopOff.java

```diff
@@ -54,7 +54,8 @@ public JSONStreamAware processRequest(HttpServletRequest req) {
         List<? extends Block> blocks;
         BlockchainProcessor blockchainProcessor = lookupBlockchainProcessor();
         try {
-            blockchainProcessor.setGetMoreBlocks(false);
+            blockchainProcessor.suspendBlockchainDownloading();
+            blockchainProcessor.waitUntilBlockchainDownloadingStops();
```

Comment:
Its not necessary, because GlobalSync.writeLock in popOff guarantee that blockchain will not being downloading

Reply:
done